http_check/README.md

```diff
@@ -49,6 +49,7 @@ See the [sample http_check.d/conf.yaml][3] for a full list and description of av
 | `tls_ignore_warning`             | If `tls_verify` is set to `true`, it disables any security warnings from the SSL connection.                                                                                                                                                     |
 | `tls_ca_cert`                    | This setting allows you to override the default certificate path as specified in `init_config`                                                                                                                                                   |
 | `check_certificate_expiration`   | When `check_certificate_expiration` is enabled, the service check checks the expiration date of the SSL certificate. Note that this causes the SSL certificate to be validated, regardless of the value of the `disable_ssl_validation` setting. |
+| `disable_ssl_validation`   | Disables HTTPS certificate validation. If set to `true` this will skip the validation of the SSL certificate of the URL being tested. |
```

Comment:
```suggestion
| `disable_ssl_validation`   | Disables HTTPS certificate validation. If set to `true` this will skip the validation of the SSL certificate of the URL being tested. |
```
```suggestion
| `disable_ssl_validation`   | Disables HTTPS certificate validation. If set to `true` this skips the validation of the SSL certificate of the URL being tested. |
```